### PR TITLE
Update Oak Item Quest Descriptions

### PR DESCRIPTION
**Pull request by CorgiOnNeptune**
Quick description update to Oak Item Quests, per Issue #2134 today.

Let me know if there is a more succinct way to write this. I'm still quite new to JS so anything new to learn is a plus.

### CODE REVIEW

**Review by RedSparr0w — CHANGES_REQUESTED** [review]
Looks good, but could we change the descriptions to read more like:
`Equip the x and capture y Pokémon`
`Equip the x and earn y PokéDollars`

**Review by RedSparr0w — APPROVED** [review]
Looks good, thanks for the PR! 👍

File: src/scripts/quests/questTypes/UseOakItemQuest.ts [hunk 1]
```diff
@@ -41,7 +41,18 @@ class UseOakItemQuest extends Quest implements QuestInterface {
...
 
     get description(): string {
-        return `Gain the benefit from the ${GameConstants.humanifyString(OakItemType[this.item])} ${this.amount.toLocaleString('en-US')} times.`;
+        const desc = [];
+        desc.push(`Equip the ${GameConstants.humanifyString(OakItemType[this.item])} and`);
+        if (OakItemType[this.item].match('Magic_Ball')) {
+            desc.push(`capture ${this.amount.toLocaleString('en-US')} wild Pokémon.`);
+        } else if (OakItemType[this.item].match('Amulet_Coin')) {
+            desc.push(`earn ${this.amount.toLocaleString('en-US')} PokéDollars.`);
+        } else if (OakItemType[this.item].match('Exp_Share')) {
```
Comment (RedSparr0w) [inline src/scripts/quests/questTypes/UseOakItemQuest.ts:50]: Sorry, 1 last thing, could you compare the values against the enum:
```js
if (this.item == OakItemType.Blaze_Cassette) {
```

Comment (CorgiOnNeptune) [reply src/scripts/quests/questTypes/UseOakItemQuest.ts]: For sure!

**Pull request merged**